code/modules/mob/living/silicon/robot/drone/drone.dm

```diff
@@ -193,6 +193,7 @@
 	emagged = 1
 	density = 1
 	pass_flags = 0
+//	icon_state = "[icon_state]-emagged"	//in case custom emagged drones are allowed, comment out the line bellow and uncomment this line
```

Comment:
I don't think you should be adding in commented out code for the sake of the future, this leads to unnecessary bloat. Just make this the default behavior and supply an emagged version of the drone